azure-pipelines.yml

```diff
@@ -1,342 +1,382 @@
-name: "${{ parameters.action }} - ${{ parameters.env }}.${{ parameters.cluster }}.${{ parameters.location }}"
+name: AKS-CFT-Deploy - ${{ parameters.action }}
 
-trigger: none
+trigger:
+  branches:
+    include:
+      - main
+
+pr:
+  branches:
+    include:
+      - refs/heads/main
 
 pool:
-  vmImage: 'ubuntu-latest'
+  vmImage: ubuntu-latest
 
 resources:
   repositories:
     - repository: cnp-azuredevops-libraries
       type: github
       ref: master
       name: hmcts/cnp-azuredevops-libraries
-      endpoint: 'hmcts'
+      endpoint: "hmcts"
     - repository: cnp-flux-config
       type: github
       ref: refs/heads/master
       name: hmcts/cnp-flux-config
-      endpoint: 'hmcts'
+      endpoint: "hmcts"
 
 parameters:
-
   - name: action
     displayName: Action
     type: string
-    default: 'plan'
+    default: plan
     values:
-    - plan
-    - apply
+      - plan
+      - apply
 
   - name: cluster
     displayName: Cluster
     type: string
-    default: '00'
+    default: All
     values:
-    - 'All'
-    - '00'
-    - '01'
+      - "All"
+      - "00"
+      - "01"
 
   - name: location
     displayName: Location
     type: string
-    default: 'UK South'
+    default: "UK South"
     values:
-    - 'UK South'
-    - 'UK West'
+      - "UK South"
 
-  - name: env
-    displayName: Environment
-    type: string
-    default: 'sbox'
-    values:
-    - ithc
-    - preview
-    - demo
-    - sbox
-    - perftest
-    - aat
-    - ptlsbox
-    - ptl
-    - prod
+  - name: environment_components
```

Comment:
maybe something more meaning?

Reply:
All other pipeline in other repo used the same name so i left it there too

Reply:
not really same context I dont think, but can leave for now